kokoro/presubmit.sh

```diff
@@ -5,21 +5,27 @@ set -o xtrace
 
 gcloud components install docker-credential-gcr
 
+# Docker service does not run by default in Big Sur but can be started with the following commands.
+if [ "${KOKORO_JOB_CLUSTER}" = "MACOS_EXTERNAL" ]; then
+docker-machine ls
+docker-machine start default
+docker-machine ssh default "echo '{ \"insecure-registries\":[\"192.168.99.104:5000\"] }' | sudo tee /etc/docker/daemon.json "
```

Comment:
In the end, this should probably be
```suggestion
docker-machine ssh default "echo '{ \"insecure-registries\":[\"$DOCKER_IP:5000\"] }' | sudo tee /etc/docker/daemon.json "
```
and come after the `export DOCKER_IP` below.

Also, 5000 is the usual port for a local registry, but I remember we also use 6000. I wonder if we can omit the port part or have a way to specify any ports.

Reply:
Good point! Referencing the env variable is definitely better in the long run. I think eventually this fix will be applied in the image itself.